src/Components/Patient/PatientRegister.tsx

```diff
@@ -1738,15 +1760,15 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                           }
                           title={
                             <h1 className="text-left text-xl font-bold text-purple-500">
-                              COVID Details
+                              Complete Vaccination Details
```

Comment:
use internationalization, we are working on adding it through out the application, so when you come across an instance where it is not used while making changes, please feel free to add